python/lsst/ts/standardscripts/maintel/setup_mtcs.py

```diff
@@ -217,6 +234,8 @@ async def start_mtmount(self):
             components=["mtmount"],
             overrides=dict(mtmount=self.config.overrides["mtmount"]),
         )
+        self.log.info("Home mtmount")
+        await self.mtcs.rem.mtmount.cmd_homeBothAxes.set_start()
```

Comment:
need to extract this into a method in ts_observatory_control.. Can you add a TODO here and issue a ticket for me to do that?


Also, maybe use `start` instead of `set_start` and add a timeout?